data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_abilities_class.lst

```diff
@@ -370,7 +375,7 @@ Draconic Resistance		KEY:Draconic Resistance ~ Acid				CATEGORY:Special Ability
+Breath Weapon			KEY:Draconic Bloodrager ~ Breath Weapon			CATEGORY:Special Ability	TYPE:Internal.Class Feature						VISIBLE:NO	DEFINE:BreathWeaponTimes|0	DEFINE:BreathWeaponDice|0	DEFINE:BreathWeaponDC|0																																																																																				ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Acid Cone|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Green)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Acid Line|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Black),Bloodrager Bloodline ~ Draconic (Copper)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Cold|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (White),Bloodrager Bloodline ~ Draconic (Silver)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Electricity|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Blue),Bloodrager Bloodline ~ Draconic (Bronze)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Fire Cone|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Red),Bloodrager Bloodline ~ Draconic (Gold)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Fire Line|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Brass)																																		BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,8	BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,16	BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,20	BONUS:VAR|BreathWeaponDice|BloodlineLVL	BONUS:VAR|BreathWeaponDC|10+(BloodlineLVL/2)+CON
```

Comment:
Dragon Disciple can give you access to it earlier than level 8, so if you get it at 7 the formula gives you an extra use. Before you had really weird behavior with this where you'd get the bloodrager breath weapon and the sorcerer breath weapon (and doubled up on the damage on both), but the sorcerer breath weapon had uninitialized variables and incomplete text descriptions. The PRECLASS in the core rulebook is one of the strangest things I encountered so far, because it doesn't give you any errors at all. Loading Core Rulebook only, gaining the ability that has the tag, everything works fine, I don't even get a warning (this might actually be a bug). I don't know if there is another way to block bloodragers from getting this without prohibiting it in the core rulebook, other than adding the ability again in the ACG with bloodrager in mind.